dspace-api/src/main/java/org/dspace/app/itemupdate/AddBitstreamsAction.java

```diff
@@ -210,7 +210,7 @@ protected String addBitstream(Context context, ItemArchive itarch, Item item, Fi
 				if (group != null)
 				{
                     authorizeService.removeAllPolicies(context, bs);  // remove the default policy
-                    authorizeService.createResourcePolicy(context, bs, group, null, ce.permissionsActionId, null);
+                    authorizeService.createResourcePolicy(context, bs, group, null, ce.permissionsActionId, null, null, null, null, null);
```

Comment:
This call looks very odd (with 5 `null` values passed in as the last five attributes).  I wonder if we should simply have multiple `createResourcePolicy()` methods with different numbers of inputs, so that we don't have to pass in such large numbers of `null` inputs.